src/modules/users/application/dto/user.response.dto.ts

```
@@ -1,7 +1,14 @@
 import { ApiProperty } from '@nestjs/swagger';
-import { IsString, IsUUID, IsEmail, IsBoolean, IsDate } from 'class-validator';
-
+import {
+  IsString,
+  IsUUID,
+  IsEmail,
+  IsBoolean,
+  IsDate,
+  IsOptional,
```

ESLint

[error] 'IsOptional' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)